File: Assets/VirtualCity/ProtoDefine/Java/system/ReqCheckCommendCodeMessage.java
package com.kingston.jforgame.server.game.system.message;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.system.SystemDataPool;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

@MessageMeta(module = Modules.SYSTEM,cmd = SystemDataPool.REQ_CKCODE)
public class ReqCheckCommendCodeMessage extends Message {
    @Protobuf(order = 1)
    private String code;

    public ReqCheckCommendCodeMessage() {
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public ReqCheckCommendCodeMessage(String code) {
        this.code = code;
    }
}

File: Assets/VirtualCity/ProtoDefine/ReqCheckCommendCodeMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]


public class ReqCheckCommendCodeMessage {
[ProtoMember(1)]
    public string code;

    public ReqCheckCommendCodeMessage() {
    }

    public string getCode() {
        return code;
    }

    public void setCode(string code) {
        this.code = code;
    }

    public ReqCheckCommendCodeMessage(string code) {
        this.code = code;
    }
}
}